nbproject/project.properties

```diff
@@ -0,0 +1,4 @@
+file.reference.ophelia-public=public
+files.encoding=UTF-8
+site.root.folder=${file.reference.ophelia-public}
+source.folder=
```

Comment:
These look like files related to your IDE, so make sure you aren't committing these. You can add a rule in the `.gitignore` file to exclude these.